app/assets/stylesheets/scaffolds.scss

```diff
@@ -0,0 +1,65 @@
+body {
+  background-color: #fff;
+  color: #333;
+  margin: 33px; }
```

Comment:
Optional
'}' - should be on a new line
PS: everywhere
Example:
`.ypaAdElement {
	display: block !important;
	padding-bottom: 20px !important;
}`